src/shared/clients/codecatalystClient.ts

```diff
@@ -680,17 +681,25 @@ class CodeCatalystClientInternal {
                 const lastStatus = statuses[statuses.length - 1]
                 const elapsed = Date.now() - lastStatus.start
                 const resp = await this.getDevEnvironment(args)
+                const serviceReason = (resp.statusReason ?? '').trim()
                 alias = resp.alias
 
                 if (
-                    lastStatus &&
+                    startAttempts > 2 &&
+                    elapsed > 10000 &&
                     ['STOPPED', 'FAILED'].includes(lastStatus.status) &&
-                    ['STOPPED', 'FAILED'].includes(resp.status) &&
-                    elapsed > 60000 &&
-                    startAttempts > 2
+                    ['STOPPED', 'FAILED'].includes(resp.status)
```

Comment:
Only wait 10 seconds instead of 1 minute before deciding that a stopped/failed devenv is not going to recover.